src/wallet/bdb.cpp

```diff
@@ -803,3 +803,67 @@ std::string BerkeleyDatabaseVersion()
 {
     return DbEnv::version(nullptr, nullptr, nullptr);
 }
+
+bool BerkeleyBatch::ReadKey(CDataStream& key, CDataStream& value)
+{
+    if (!pdb)
+        return false;
+
+    // Key
+    SafeDbt datKey(key.data(), key.size());
+
+    // Read
+    SafeDbt datValue;
+    int ret = pdb->get(activeTxn, datKey, datValue, 0);
```

Comment:
Maybe just early return` false` here if `ret != 0 || datValue.get_data() == nullptr`?

Reply:
Meh. Will change if another push is needed.